test/format/CMakeLists.txt

```diff
@@ -6,20 +6,20 @@ set(format_sources
     bulk.c
     checkpoint.c
     compact.c
-    config.c
+    form_config.c
```

Comment:
Nitpick: can you make the `form_` ones `format_` instead please?

Reply:
Conflicting files listed with occurrence and updated paths below:

- `config.c `- 3 instances
```
src/config/config.c
bench/wtperf/wtperf_config.c
test/format/format_config.c
```

- `file.c` - 2 instances
```
test/thread/file.c
test/fops/fops_file.c
```

- `config_def` - 2 instances
```
src/config/config_def.c
test/format/format_config_def.c
```

- `salvage.c `- 2 instances
```
test/salvage/salvage.c
test/format/format_salvage.c
```

- `timestamp.c `- 2 instances
```
src/support/timestamp.c
test/format/format_timestamp.c
```

- `util.c `- 2 instances
```
test/csuite/random_directio/util.c
test/format/format_util.c
```

- `misc.c` - 2 instances
```
bench/wtperf/misc.c
test/utility/util_miscs.c
```

- `modify.c `- 2 instances
```
src/support/modify.c
test/utility/util_modify.c
```

- `random.c` - 2 instances
```
test/format/random.c
test/utility/util_random.c
```

Reply:
Thanks, this is super useful! Given that we already have a `util_misc.c`, would it be better to rename `wtperf/misc.c` to `wtperf/wtperf_misc.c` and leave `test/utility/misc.c` alone?